restful_api/data/ir_config_param.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+        <record id="access_token_expires_in" model="ir.config_parameter">
+            <field name="key">access_token_expires_in</field>
+            <field name="value">31536000</field>
```

Comment:
The access token is valid for 1 year?

Reply:
Yep 1 year.

Reply:
Seems too long. Why is that? Can you elaborate a bit more, about why this token is needed, if its going to be valid for a year? Usually tokens don't span more than 1-24 hours.. but maybe this is a different usecase.